src/rpc/core_rpc_server.cpp

```diff
@@ -927,6 +1021,10 @@ namespace cryptonote
   bool core_rpc_server::on_getblockhash(const COMMAND_RPC_GETBLOCKHASH::request& req, COMMAND_RPC_GETBLOCKHASH::response& res, epee::json_rpc::error& error_resp)
   {
     PERF_TIMER(on_getblockhash);
+    // if (should_use_bootstrap_daemon())
```

Comment:
Why is this not active?

Reply:
See https://github.com/monero-project/monero/pull/3165#discussion_r162804653